tools/cli/go-whisk-cli/commands/action.go

```diff
@@ -121,7 +121,7 @@ var actionInvokeCmd = &cobra.Command{
     RunE: func(cmd *cobra.Command, args []string) error {
         var err error
         var parameters interface{}
-        var qualifiedName QualifiedName
+        var qualifiedName = new(QualifiedName)
```

Comment:
Please go through all of the command files and make sure the proper replacements have been made for the QualifiedName functions that you have added.

`qualifiedName.packageName` line 235
`printActionGetWithURL(qualifiedName.entity, actionURL)` line 236